content/swagger/parameters.yaml

```
@@ -85,3 +85,10 @@ with_count:
   default: false
   type: boolean
   required: false
+with_attribute_options:
```

The naming of this query parameter does not seem really clear as we already return by default the attribute options. But the code, not the label. So I would suggest:
```suggestion
with_attribute_option_labels:
```
@meganelepalud-akeneo What do you think?